Tests/SwiftStdlibTests/RangeReplaceableCollectionExtensionsTests.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  RangeReplaceableCollectionExtensionsTests.swift
+//  SwifterSwift
+//
+//  Created by Max Härtwig on 12.06.18.
+//  Copyright © 2018 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+final class RangeReplaceableCollectionExtTests: XCTestCase {
```

Comment:
The class name should match the file name.

Reply:
I agree, but swiftlint doesn't allow class names longer than 40 characters causing the Travis build to fail.

Reply:
You can hint to swiftlint to ignore a specific rule for a specific line by adding a comment above it like this:
`swiftlint:disable next identifier_name`
I'm not sure if `identifier_name` is the exact rule that's being broken by the long class name, but it's something like that.

Reply:
I believe we already have this file and class created on master, it was created in #512 if I'm remembering correctly :)

Reply:
Ah true, and the class name ended up being `RangeReplaceableCollectionTests`, I'm guessing to avoid that issue.